packages/editor/src/plugins/image/utils/check-image-url.ts

```diff
@@ -0,0 +1,6 @@
+const imageRegex =
+  /^(https?:\/\/(?:www\.)?[\w.-]+\.[a-z]{2,}(?:\/[\w.-]*)*\.(?:jpg|jpeg|png|gif|bmp|tiff|webp|svg))$/
+
+export const isImageUrl = (url: string) => {
+  return imageRegex.test(url)
+}
```

Comment:
Could this be reused here as well: https://github.com/serlo/frontend/blob/staging/packages/editor/src/plugins/image/index.ts#L69 ?

On first glance, it seems like the two regex strings are doing the same thing, but it might be good to have a deeper look into the code to check if they really are the same.

Reply:
The difference is the requirement for the http: or https, which I dont think should be necessary, so I would move  the original regex to this util and use the util function in both places

Reply:
The first regex matches an entire URL including the protocol, domain, and path, ensuring it is an image URL.
The second regex only matches the file extension at the end of a string.

@elbotho @hejtful which do you think we should be using?

Reply:
Just saw the thumbs up, nvm

Reply:
Done